<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OneMonitorWidgetClass</class>
 <widget class="QWidget" name="OneMonitorWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OneMonitorWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame
{
background-color:rgb(50,50,50);
}

QWidget#widget
{
	background-color:rgb(65,65,65);
	border-radius:8;
	border:2px solid #4F4F4F;
}

QLabel
{
	color:white;
	font-family:KaiTi;
	background: &quot;transparent&quot;;
	border:none;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,6">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_ControlVisiable">
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>18</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	border-image:url(&quot;:/DynamicWidget/Resource/GPU_Open.png&quot;);
	border:none;
}
QPushButton:hover
{
	background-color:rgb(10,10,10);
}
QPushButton:pressed
{
	background-color:rgb(35,35,35);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_GPUID">
          <property name="font">
           <font>
            <family>KaiTi</family>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>GPU1</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>KaiTi</family>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Performance:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Label_PerformanceWord">
             <property name="font">
              <font>
               <family>KaiTi</family>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>UnKonw</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_Line1">
             <property name="minimumSize">
              <size>
               <width>2</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>2</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image:url(&quot;:/DynamicWidget/Resource/ToolBarSegLine.png&quot;);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>KaiTi</family>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Temperature:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Label_TemperatureWord">
             <property name="font">
              <font>
               <family>KaiTi</family>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>UnKonw</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_Line2">
             <property name="minimumSize">
              <size>
               <width>2</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>2</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image:url(&quot;:/DynamicWidget/Resource/ToolBarSegLine.png&quot;);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>KaiTi</family>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Utilization:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Label_UtilizationWord">
             <property name="font">
              <font>
               <family>KaiTi</family>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>UnKonw</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_Line3">
             <property name="minimumSize">
              <size>
               <width>2</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>2</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image:url(&quot;:/DynamicWidget/Resource/ToolBarSegLine.png&quot;);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>KaiTi</family>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Power:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Label_PowerWord">
             <property name="font">
              <font>
               <family>KaiTi</family>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>UnKonw</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>KaiTi</family>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>GPU Memory</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="progressBar_MemoryBar">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>12</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>10</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QProgressBar
{
		color:yellow;
		text-align: center;
		background: rgba(248,248,255,180);
		border-radius: 6px;
}
QProgressBar::chunk:enabled 
{
		border-radius: 6px; 
		background: qlineargradient(x1:0, y1:0, x2:1, y2:0 ,stop:0 #00AB00, stop:1 #BF0000);
}</string>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Label_MemoryNumWord">
             <property name="font">
              <font>
               <family>KaiTi</family>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>UnKonw</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
